packages/serverless/src/domain/use-case/document.ts

```diff
@@ -102,19 +100,14 @@ async function findDocument(
     );
   }
 
-  return {
-    ...documentRecord,
-    document: Object.values(
-      ModelDocument.deserializeDocument(documentRecord.document)
-    ),
-  };
+  return documentRecord;
 }
 
 async function createDocument(
```

Comment:
also you can put it to a class and use static method to perform these document thing, it will be cleaner

Reply:
added an TODO to fix this later